lib/cookbook-omnifetch/chef_server_artifact.rb

```diff
@@ -45,7 +45,8 @@ def install
     #
     # @return [Boolean]
     def installed?
```

Comment:
is the point in keeping this just backwards compatibility?

Reply:
Yes

Reply:
Can you add yard deprecated comments on those too then

Reply:
Backing up - this change was not for back-compat. 

This is the mechanism that we use to determine if a cookbook is cached, via https://github.com/chef/chef-cli/blob/master/lib/chef-cli/policyfile/cookbook_location_specification.rb#L75-L84.  This change was to have these two report as not cached, so that in the refresh we could handle it with per-file caching instead of per-cookbook. 


It's a required implementation inherited from the base class  `CookbookOmnifetch::BaseLocation`.